src/Physics/Evaluate_friction_law.f90

```diff
@@ -57,8 +57,16 @@ MODULE Eval_friction_law_mod
   INTERFACE Eval_friction_law
      MODULE PROCEDURE Eval_friction_law
   END INTERFACE
+  !INTERFACE update_RSF
+     !MODULE PROCEDURE update_RSF
+  !END INTERFACE
+    !INTERFACE Newton_Raphson
+     !MODULE PROCEDURE Newton_Raphson
+  !END INTERFACE
   !---------------------------------------------------------------------------!
   PUBLIC  :: Eval_friction_law
+  PRIVATE :: update_RSF
+  PRIVATE :: Newton_Raphson
```

Comment:
These names are not sensible:
- RSF is an unnecessary abbreviation and breaks with the rate_and_state naming scheme.
- Both names suggest to be more general then they actually are. In particular Newton_Raphson is a method that may be applied to many problems,

Reply:
sorry but this is not resolved.
What about updateStateVariable instead of update_RSF
and IterativelyInvertSR instead of Newton_Raphson?